xml/System.Net.Sockets/Socket.xml

```diff
@@ -4678,21 +4678,27 @@ This method populates the <xref:System.Net.Sockets.Socket> instance with data ga
         <remarks>
           <format type="text/markdown"><![CDATA[  
   
-## Remarks  
- The target process should use <xref:System.Net.Sockets.Socket.%23ctor%2A> to create the duplicate socket instance.  
-  
- If you call the <xref:System.Net.Sockets.Socket.%23ctor%2A> constructor multiple times with the same byte array as the argument for each call, you will create multiple managed <xref:System.Net.Sockets.Socket> instances with the same underlying socket. This practice is strongly discouraged.  
-  
- If the process creating the socket uses asynchronous methods (<xref:System.Net.Sockets.Socket.BeginReceive%2A> or <xref:System.Net.Sockets.Socket.BeginSend%2A>), the process must first set the <xref:System.Net.Sockets.Socket.UseOnlyOverlappedIO%2A> property to true; otherwise, the socket is bound to the completion port of the creating process, which may cause an <xref:System.ArgumentNullException> to be thrown on the target process.  
+## Remarks
+
+The target process should use <xref:System.Net.Sockets.Socket.%23ctor%2A> to create the duplicate socket instance.
```

Comment:
```suggestion
The target process should use the <see cref="M:System.Net.Sockets.Socket.#ctor(System.Net.Sockets.SocketInformation)" /> constructor to create the duplicate socket instance.
```

Reply:
The reference links have to be in xref format for the XML files.